doc/ss_logger.rst

```diff
@@ -0,0 +1,96 @@
+********
+Overview
+********
+The SmartSim library provides logging functionality that users can quickly
+integrate into an experiment. A SmartSim logger helps debug your experiment
+and can help keep track of the experiment flow. On this page, we demonstrate
+how to setup the a SmartSim logger within a python script.
+
+First, import the `get_logger` module.
+Initialize a logger instance by providing a `name` to the function ``get_logger()``:
+
+.. code-block:: python
+
+      from smartsim.log import get_logger
+
+      logger = get_logger("example_logger")
+
+There are three logging levels indicating the severity of log messages.
+Each log level is associated with a function that can be used to log events at
+the associated level. The log levels, in order of increasing
+severity, are as follows:
+
+1. quiet
+2. info
+3. debug
+
+Next, we place all four logging functions within the Python script:
+
+.. code-block:: python
+
+      logger.info("This is a message")
+      logger.debug("This is a debug message")
+      logger.error("This is an error message")
+      logger.warning("This is a warning message")
+
+When we run the script, the following output prints to stdout::
+    19:52:05 osprey.us.cray.com SmartSim[130033] INFO This is a message
+    19:52:05 osprey.us.cray.com SmartSim[130033] ERROR This is an error message
+    19:52:05 osprey.us.cray.com SmartSim[130033] WARNING This is a warning message
+
+Notice that the `debug` function did not print. This is because
+`SMARTSIM_LOG_LEVEL` is defaulted to `info`. The `SMARTSIM_LOG_LEVEL` environment
+variable controls what log level messages are printed. Given a log level, SmartSim will
+ignore higher severity levels if present. Such as here, since `debug` is higher than `info`,
+it is ignored.
```

Comment:
Maybe change 'higher' to 'more verbose' or 'more severe' or whichever descriptor you choose.